src/lib.rs

```diff
@@ -77,6 +83,7 @@ where
     W: Write,
 {
     run_with_archive(
+        Options::IgnoreOwnership,
```

Comment:
I don't think it goes for a clean code to have ownership argument being used on the `uncompress_file` function. How about we store it inside the `Mode::WriteDisk` enum value, since it's the only case that needs it?

Reply:
I think this is a good suggestion